application_code/client_apps/java/pom.xml

```diff
@@ -48,6 +48,7 @@
         <module>s4k_kafka_consumer</module>
         <module>s4r_rabbitmq_queue_consumer</module>
         <module>s4r_rabbitmq_queue_producer</module>
+        <module>function_astradb_v1</module>
```

Comment:
Can you please remove this from here (for client applications) and move it to "functions" side?